### PR TITLE
Add recycler view

### PR DESCRIPTION
**Pull request by melissa-vuz**
Added the second recycler view but it doesn't save data, because I'm not sure how I want to handle that data yet.

### CODE REVIEW

File: app/src/main/java/com/example/myfirstapp/ShoppingTabFragment.java [hunk 1]
```diff
@@ -2,16 +2,121 @@
...
 
 /**
  */
 public class ShoppingTabFragment extends Fragment {
+
+    // Adapter
+    SimpleAdapter recyclerAdapter;
+
+    // RecyclerView
+    RecyclerView recyclerView;
```
Comment (rf43) [inline app/src/main/java/com/example/myfirstapp/ShoppingTabFragment.java:27]: It's best to keep the visibility as low as possible. So, you might consider making these private as opposed to package-private.

File: app/src/main/java/com/example/myfirstapp/ShoppingTabFragment.java [hunk 2]
```diff
@@ -2,16 +2,121 @@
...
+    // Button for adding item
+    Button addItem;
+
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle bundle) {
-        return inflater.inflate(R.layout.fragment_shopping_tab, container, false);
+        super.onCreate(bundle);
+        final View view = inflater.inflate(R.layout.fragment_shopping_tab, container, false);
+
+        editShoppingItemName = view.findViewById(R.id.editItemName);
```
Comment (rf43) [inline app/src/main/java/com/example/myfirstapp/ShoppingTabFragment.java:46]: Do you want to keep things like this "pure" or are you open to library like [Butter Knife](http://jakewharton.github.io/butterknife/)?

Comment (melissa-vuz) [reply app/src/main/java/com/example/myfirstapp/ShoppingTabFragment.java:46]: I don't know what that difference would be between keeping it "pure" or using Butter Knife. Though Butter Knife is on my list of things to learn so let's do that.

File: app/src/main/java/com/example/myfirstapp/ShoppingTabFragment.java [hunk 3]
```diff
@@ -2,16 +2,121 @@
...
+
+            @Override
+            public void onTextChanged(CharSequence charSequence, int i, int i1, int i2) {
+
+            }
+
+            @Override
+            public void afterTextChanged(Editable editable) {
+                editShoppingItemPrice.removeTextChangedListener(this);
+                editShoppingItemPrice.setText("$ " + editShoppingItemPrice.getText().toString());
```
Comment (rf43) [inline app/src/main/java/com/example/myfirstapp/ShoppingTabFragment.java:69]: You can make that `$` a string res and then you can use `String.format()`

Your string res might look something like:
`<string name="money_prefix">$ %s</string>`


and the String.format would look something like this:
`String.format(Locale.US, getString(R.string.money_prefix), editShoppingItemPrice.getText().toString())`

**Review by rf43 — APPROVED** [review]

File: app/src/main/java/com/example/myfirstapp/ShoppingTabFragment.java (hunk 2 above)
Comment (rf43) [reply app/src/main/java/com/example/myfirstapp/ShoppingTabFragment.java:46]: word!

**Pull request merged**